src/components/FindWallet/WalletPersonasSidebar.tsx

```diff
@@ -30,157 +36,6 @@ import Eip1559 from "../../assets/wallets/eip1559.svg"
 // Utils
 import { trackCustomEvent } from "../../utils/matomo"
 
-// Styles
-const Container = styled.div`
-  display: flex;
-  flex-direction: column;
-  gap: 16px;
-
-  @media (max-width: ${(props) => props.theme.breakpoints.s}) {
-    padding: 1rem 1rem;
-  }
-`
-
-const FeatureListItem = styled.li<{
-  selected: boolean
-}>`
-  display: flex;
-  gap: 0.2rem;
-  font-size: 0.85rem;
-  line-height: 0.95rem;
-  margin: 0.1rem;
-  align-items: center;
-  p {
-    margin-bottom: 0;
-    color: ${(props) =>
-      props.selected === true
-        ? props.theme.colors.primary
-        : props.theme.colors.text};
-  }
-
-  svg {
-    width: 28px;
-    height: 28px;
-    path {
-      fill: ${(props) => props.theme.colors.text};
-      stroke: ${(props) => props.theme.colors.text};
-    }
-  }
-`
-
-const Persona = styled.div<{
-  selected: boolean
-  isDark: boolean
-}>`
-  display: flex;
-  flex-direction: column;
-  align-items: flex-start;
-  padding: 1.5rem;
-  background: ${({ selected, isDark, theme }) =>
-    selected
-      ? isDark
-        ? theme.colors.primary900
-        : theme.colors.primary200
-      : isDark
-      ? theme.colors.black400
-      : theme.colors.primary100};
-  border-radius: 4px;
-  cursor: pointer;
-  transition: 0.5s all;
-
-  h3 {
-    color: ${(props) => props.theme.colors.text};
-  }
-
-  &:hover {
-    background: ${(props) =>
-      props.selected === true
-        ? props.isDark === true
-          ? props.theme.colors.primary900
-          : props.theme.colors.primary200
-        : props.isDark === true
-        ? props.theme.colors.black500
-        : props.theme.colors.primary200};
-    transition: 0.5s all;
-  }
-`
-
-const PersonaDescription = styled.span<{
-  selected: boolean
-}>`
-  margin: 0.5rem 0 0.8rem 0;
-  padding: 0.7rem 0.6rem 0;
-  color: ${(props) =>
-    props.selected === true
-      ? props.theme.colors.text
-      : props.theme.colors.text200};
-  font-size: 0.9rem;
-  font-weight: normal;
-  transition: 0.5s all;
-  line-height: 1.3;
-  border-top: 1px solid ${(props) => props.theme.colors.lightBorder};
-`
-
-const Title = styled.div`
-  display: flex;
-  gap: 0.5rem;
-  align-items: center;
-  gap: "1rem";
```

Comment:
There are two gap values. Which one should we choose?

Reply:
Good catch. `gap: 0.5rem` is currently applied on prod. Lets use that.